### DISCUSSION TITLE
Copy marked files?

### DISCUSSION THREAD

**Discussion by baronyoung** [nvim-tree/nvim-tree.lua#2842] [Q&A] [answered]
This seems like it must be a dumb question because it's such a common need, but I can't find any mention of how to copy marked files.  I can easily copy a single file I have highlighted but I can't find any mention in the help document about how I can copy all the files I have marked.

**Comment by alex-courtis** [comment, marked as answer]
You can bulk move, but not copy.

See `:help nvim-tree-quickstart-help` for mappings: `bmv` in this case.

Bulk copy could be added, however it would need to be implemented by a community contributor due to time constraints: [ONTRIBUTING.md](https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md)

**Reply by baronyoung** [reply]
Ok, got it.  I'm sure I can figure out a workaround.  Thanks for the reply.

**Reply by alex-courtis** [reply]
If you do, I'd be most grateful for a [Recipe](https://github.com/nvim-tree/nvim-tree.lua/wiki/Recipes) or PR

**Reply by Frankwii** [reply]
Hi! I got it to work!! Added a recipe for it.
